lib/mutant/meta/example/verification.rb

```diff
@@ -0,0 +1,86 @@
+module Mutant
+  module Meta
+    class Example
+      # Example verification
+      class Verification
+        include Adamantium::Flat, Concord.new(:example, :mutations)
+
+        # Test if mutation was verified successfully
+        #
+        # @return [Boolean]
+        def success?
+          missing.empty? && unexpected.empty? && no_diffs.empty?
+        end
+
+        # Error report
+        #
+        # @return [String]
+        def error_report
+          fail 'no error report on successful validation' if success?
```

Comment:
You're ok with this failing with a `StandardError` exception? Not something more specific?